src/core/domain/userdm/user_domain_service.go

```diff
@@ -0,0 +1,19 @@
+package userdm
+
+import "context"
+
+type UserDomainService struct {
+	userRepository UserRepository
+}
+
+func NewUserDomainService(userRepo UserRepository) *UserDomainService {
+	return &UserDomainService{userRepository: userRepo}
+}
+
+func (ds *UserDomainService) IsExists(ctx context.Context, userID string) (bool, error) {
+	_, err := ds.userRepository.FindByID(ctx, userID)
+	if err != nil {
+		return false, err
+	}
+	return true, nil
```

Comment:
```suggestion
	user, err := ds.userRepository.FindByID(ctx, userID)
	if err != nil {
		return false, err
	}
	return user != nil, nil
```

FindByIDの一個目の返却値がnilであることも一応チェックしましょう。

Reply:
上記の通りに修正しました。